packages/frontend/src/components/ExportCSV/ExportCSVModal.tsx

```diff
@@ -55,7 +55,7 @@ const ExportCSVModal: FC<ExportCSVModalProps> = ({
                         <Button
                             loading={isExporting}
                             onClick={() => {
-                                onExport().then(() => {
+                                void onExport().then(() => {
```

Comment:
error can be ignored? maybe not?

Reply:
Maybe out of scope too, but using a floating promise + `then` on a button seems a bit fishy - I'd make a note to either block on this (with `await`):

```ts
onClick={async () => {
  await onExport();
  onConfirm?.();
}
```

...or implement it within the component render state somehow.